src/main/java/com/example/pupbuddy/dto/Chore.java

```diff
@@ -4,11 +4,11 @@
 
 @Data
 public class Chore {
-    private int ChoreId;
-    private String choreName;
-    private boolean choreComplete;
-    private String choreStart;
-    private String choreEnd;
+    private int choreId;
+    private String name;
+    private boolean complete;
+    private String start;
+    private String end;
```

Comment:
I renamed these variables to be more consistent with other classes, which don't use the class name as a prefix except for IDs. Additionally, variables shouldn't start with a capital letter but a lowercase one (see [Java Naming Conventions](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) from Oracle).

Reply:
Good.

Reply:
As a group we are going with a naming convention that is more consistent over each of our classes, for example it will be easier to distinguish choreName from petName in a large code block.